### DISCUSSION TITLE
wired_table_rec_v2也是读光的模型吗？

### DISCUSSION THREAD

**Discussion by vux979** [RapidAI/TableStructureRec#64] [Q&A]
感觉wired_table_rec_v1和魔搭上读光的有线识别效果差不多，但是wired_table_rec_v2的效果更好。想知道wired_table_rec_v2是哪个模型。

**Comment by Joker1212** [comment]
不是，是别人贡献的线框模型，精度比读光的高很多

**Reply by vux979** [reply]
这个模型是开源的吗？我在您的仓库中没有找到对应的文件。如果是开源的能否提供权重和对应文件，或者相关的链接，非常感谢。

**Reply by Joker1212** [reply]
是个人贡献的，在release里面，因为他本意是想提供有偿服务，所以这里只有onnx格式，不支持微调再训练，但是如果你希望重新训练的话，我猜想可以使用unet网络进行训练，参考chineseocr项目

**Reply by Athena233** [reply]
> 不是，是别人贡献的线框模型，精度比读光的高很多

你好博主，我在尝试获取onnx文件，我先运行了setup_wired.py，得到了wheel文件，安装wheel后并没有onnx的文件的生成，我到底是哪里出错了呢？

**Reply by Joker1212** [reply]
setup脚本是把本地有的东西打包成whl，也就是说打包前本地已经有onnx了，如果你需要这个模型，可以直接在这里找到哈
https://github.com/RapidAI/TableStructureRec/releases/tag/v0.0.0

> > 不是，是别人贡献的线框模型，精度比读光的高很多
> 
> 你好博主，我在尝试获取onnx文件，我先运行了setup_wired.py，得到了wheel文件，安装wheel后并没有onnx的文件的生成，我到底是哪里出错了呢？

**Reply by Athena233** [reply]
多谢博主！
